core/tests/protractor/collections.js

```diff
@@ -17,30 +17,29 @@
  */
 
 var general = require('../protractor_utils/general.js');
-var users = require('../protractor_utils/users.js');
+var UsersPage = require('../protractor_utils/UsersPage.js');
```

Comment:
See @seanlip's comment, I think you mean to say "UsersPreferencesPage" here, we don't have a "Users" page in Oppia!